modules/internal/NetworkAtomics.chpl

```diff
@@ -32,6 +32,17 @@ module NetworkAtomics {
   record RAtomicBool {
     var _v: int(64);
 
+    pragma "no doc"
+    proc init=(other : RAtomicBool) {
+      this._v = other._v;
```

Comment:
I wonder if this should be `this._v = other.read():int(64)` (if we need/want the read of other to be atomic)

Reply:
(Initializing the current atomic is _not_ atomic, but I'm not sure if we need the read of other to be atomic)